scripts/install

```diff
@@ -1,12 +1,17 @@
-#!/bin/sh -e
+#!/usr/bin/env bash
+
+set -exo pipefail
```

Comment:
I reckon we should probably omit this. The `pipefail` is extraneous here, and would have already included `-x` if had wanted that.

We could potentially use `set -x` immediately prior *running* to the pip install commands, but let's treat that all as a separate PR and try to keep this as absolutely isolated as possible to "what do we need to do in order to support running the tests on windows".